app/models/manager_refresh/inventory/core.rb

```diff
@@ -0,0 +1,41 @@
+module ManagerRefresh::Inventory::Core
```

Comment:
What is the benefit of having everything in `Core` and also in their respective managers?  You could use modules to reduce the repetition but why bother?